Scripts/Core/EditorVR.Viewer.cs

```diff
@@ -197,7 +197,12 @@ internal void AddPlayerModel()
                 m_PlayerBody = ObjectUtils.Instantiate(evr.m_PlayerModelPrefab, CameraUtils.GetMainCamera().transform, false).GetComponent<PlayerBody>();
                 var renderer = m_PlayerBody.GetComponent<Renderer>();
                 evr.GetModule<SpatialHashModule>().spatialHash.AddObject(renderer, renderer.bounds);
-                renderer.GetComponentsInChildren(true, m_VRPlayerObjects);
+                var playerObjects = m_PlayerBody.GetComponentsInChildren<Renderer>(true);
```

Comment:
really these are `playerRenderers`, you get objects attached to them, but not a big deal